### ISSUE TITLE
Underline the importance of "business needs" in Phase 0

### ISSUE THREAD

**Issue by lorenzomassacci** [e-xtrategy/architectural-clash#5]
I think the actual version of [Phase 0](https://github.com/e-xtrategy/architectural-clash#phase-0-choose-the-task) doesn't explain the importance of **business needs** and importance of doing this Phase together business people and project stakeholders.

What do you think about it?

**Comment by francesco-strazzullo** [comment]
You're right, I can grab something from my post on [medium](https://medium.com/@TheStrazz86/architectural-clash-1baa6dc1e09d). What do you think?

**Comment by lorenzomassacci** [comment]
Good!

I'm thinking that it is also useful improve [Who](https://github.com/e-xtrategy/architectural-clash#who) section saying that business people and project stakeholder are important

**Comment by francesco-strazzullo** [comment]
Ok in the next days I will write a PR and ask you to review then!
